netclient/functions/mqhandlers.go

```diff
@@ -243,9 +246,20 @@ func UpdatePeers(client mqtt.Client, msg mqtt.Message) {
 
 func setHostDNS(dns, iface string, windows bool) error {
 	etchosts := "/etc/hosts"
+	temp := os.TempDir()
+	lockfile := temp + "netclient-lock"
```

Comment:
on your reference to temp + netclient-lock you have a "/" below on line 285, as well as `lockfile = temp + "\\netclient-lock"` under the windows condition on 289 (removeHostDNS func), whereas you do not on 250-253 in the setHostDNS func. Is that a concern that you're handling the lock file differently?

Reply:
both functions should have the identical logic